neptun-cli/Cargo.toml

```diff
@@ -1,11 +1,10 @@
 [package]
-name = "boringtun-cli"
+name = "neptun-cli"
 description = "an implementation of the WireGuard® protocol designed for portability and speed"
 version = "0.6.0"
 authors = ["Noah Kennedy <nkennedy@cloudflare.com>", "Andy Grover <agrover@cloudflare.com>", "Jeff Hiner <jhiner@cloudflare.com>"]
 license = "BSD-3-Clause"
-repository = "https://github.com/cloudflare/boringtun"
-documentation = "https://docs.rs/boringtun/0.5.2/boringtun/"
```

Comment:
no more docs?

Reply:
Not until we start publishing to crates.io